projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_print.php

```diff
@@ -160,228 +173,237 @@ function formatted_page($doctype='xhtml1.1', $externalcss=true, $title='', $lang
 
 	/**
 	 * Returns the formatted CSS Code and saves it into $this->output_css and $this->output_css_plain
-	 * @param bool $plain plain text or not
-	 * @param string $default_media default @media to add to selectors without any @media
+	 *
+	 * @param bool   $plain plain text or not.
+	 * @param string $default_media default @media to add to selectors without any @media.
 	 * @access private
 	 * @version 2.0
 	 */
-	function _print($plain = false, $default_media='') {
-		if ($this->output_css && $this->output_css_plain) {
+	public function _print( $plain = false, $default_media = '' ) { // phpcs:ignore PSR2.Methods.MethodDeclaration.Underscore -- print is a reserved word anyway.
+		if ( $this->output_css && $this->output_css_plain ) {
 			return;
 		}
 
 		$output = '';
-		if (!$this->parser->get_cfg('preserve_css')) {
-			$this->_convert_raw_css($default_media);
+		if ( ! $this->parser->get_cfg( 'preserve_css' ) ) {
+			$this->convert_raw_css( $default_media );
 		}
 
 		$template = & $this->template;
 
-		if ($plain) {
-			$template = array_map('strip_tags', $template);
+		if ( $plain ) {
+			$template = array_map( 'strip_tags', $template );
 		}
 
-		if ($this->parser->get_cfg('timestamp')) {
-			array_unshift($this->tokens, array(COMMENT, ' CSSTidy ' . $this->parser->version . ': ' . date('r') . ' '));
+		if ( $this->parser->get_cfg( 'timestamp' ) ) {
+			// @todo - see if we can use GM Date.
```

Comment:
I don't see why not. This is just a comment to say when csstidy processed the file, and the only difference is what timezone the comment uses.